src/Mechanic/Mechanic.fsproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="../../paket-files/Microsoft/visualfsharp/src/fsharp/FSharp.Build/Fsc.fs">
+    <Compile Include="..\..\paket-files\Microsoft\visualfsharp\src\fsharp\FSharp.Build\Fsc.fs">
```

Comment:
This might break the build on Linux.

Reply:
Oh thanks. I reverted that but some automatic tool seems to have done it again :/

Reply:
\ are fine in MSBuild on linux